FAQs added with respect to Time phased budgets (#4996)
SumoLogic/sumologic-documentation: docs/manage/manage-subscription/usage-management.md

1. **Capacity**. You can set either query level or time-phased budgets. You can also check the query size of the last 10 queries by clicking on the **Click here** button to help you determine the appropriate size limit.
    - **Query level budgets**. Select **Query** from the **Budget Type** dropdown and enter the GB value based on your needs. Sumo Logic recommends setting a GB value per query based on the 95th percentile to stay within safe limits. 
    - **Time-phased budgets**. Select **Daily**, **Weekly**, or **Monthly** from the **Budget Type** dropdown and enter the maximum amount of budget in GB. You can set a single shared budget for an entire group by selecting the **Capacity for the Group** option, whereas, you can set a budget for individual user in the selected scope by selecting the **Capacity per User** option.
    :::note

>     - For **Daily** budgets, the capacity resets at midnight PST.

Reviewer: ```suggestion
    - For **Daily** budgets, the capacity resets every day at midnight (00:00 PST).
```

>     - For **Weekly** budgets, the capacity resets on Monday at 00:00 PST.

Reviewer: ```suggestion
    - For **Weekly** budgets, the capacity resets each Monday at 00:00 PST.
```

>     - For **Monthly** budgets, the capacity resets on the 1st day of every calendar month at 00:00 PST.

Reviewer: ```suggestion
    - For **Monthly** budgets, the capacity resets on the first day of each calendar month at 00:00 PST.
```

  - **Audit Logs**. Records the budget definition changes. Click on **View Details** to view the budget definition changes.
  - **System Audit**. Records the breaches and budget enforcement. Click on **View Details** to view the list of breaches.

## FAQs

### Handling overlapping budgets

When you configure multiple budgets for the same user, the scope with the largest capacity takes precedence. However, if the budgets are of same values (for example, Budget A and Budget B are of same value 10 GB) then the most restrictive budget takes precedence. 

> ### Exceeding the scan budget 

Author: ```suggestion
### Exceeding the scan budget query limit
```

> When you perform a log search, if the results exceed the specified capacity limit, an error message will be displayed with no results. For example, consider you set a budget of 1 GB/query, and when your log search results exceed the set budget limit, an error message will be displayed as shown below.<br/><img src={useBaseUrl('/img/manage/account/exceed-scan-budget-warning.png')} alt="exceed-scan-budget-warning" style={{border:'1px solid gray'}} width="800"/>

Author: ```suggestion
When you perform a log search, if the results exceed the specified query limit, an error message will be displayed with no results. For example, consider you set a budget of 1 GB/query, and when your log search results exceed the set budget limit, an error message will be displayed as shown below.<br/><img src={useBaseUrl('/img/manage/account/exceed-scan-budget-warning.png')} alt="exceed-scan-budget-warning" style={{border:'1px solid gray'}} width="800"/>
```
